extensions/amp-lightbox-viewer/0.1/amp-lightbox-viewer.css

```diff
@@ -68,15 +68,20 @@
   overflow: hidden !important;
 }
 
-/* Page Mask, Gallery Viewer */
+/* Page Mask, Gallery Viewer, Description Box */
 .-amp-lbv-mask,
-.-amp-lbv-gallery {
+.-amp-lbv-gallery,
+.amp-lbv-desc-box {
   position: fixed !important;
-  top:0 !important;
   left: 0 !important;
   right: 0 !important;
   bottom: 0 !important;
   background-image: none !important;
+}
+
+.-amp-lbv-mask,
```

Comment:
This is the part of css that `.amp-lbv-desc-box` doesn't share with gallery or mask. 
I am not sure the merge of `desc-box` and `gallery` `mask` is worth it.